docs/admin/user-management.rst

```diff
@@ -18,7 +18,7 @@ superusers that already exist in the CrateDB cluster. The `CREATE USER`_ and
 When CrateDB is started, the cluster contains one predefined superuser. This
 user is called ``crate``. It is not possible to create any other superusers
 
-Users cannot be backed up or restored.
+Users\' information is backed up together with the cluster\'s metadata when a snapshot is created, and it can be restored independently with the ``USERS`` option of the :ref:`sql-restore-snapshot` command.
```

Comment:
I think a backslash for escaping the apostrophe isn't needed here, it also compiles without.

And you might want to add a line break as per https://github.com/crate/crate/blob/master/devs/docs/write_docs.rst:
> Line length must not exceed 79 characters (except for literals that cannot be wrapped)